docs/administrator-guide/exporting-channels.md

```diff
@@ -1,5 +1,9 @@
-# Exporting Channels
+# Exporting Channel Logs
 
-## Exporting an ongoing incident
+You can export ongoing and ended incidents using the steps below.
 
-## Exporting an ended incident 
+1. Open **Main Menu > Playbooks & Incidents**.
+2. Select **Incidents**.
+3. Select the incident you want to export.
+4. In the top-right corner select **Export Channel Log**.
+5. Save or open the file.
```

Comment:
There's another way to export: running the slash command `/export` from within the incident's channel.